lib/SourceFilter/SourceFilter.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { AcqCheckboxFilter } from '../AcqCheckboxFilter';
+import { sourceOptions } from '../constants';
+import { selectOptionShape } from '../shapes';
+
+const SourceFilter = (props) => (
+  <AcqCheckboxFilter
+    {...props}
```

Comment:
I think you can put here label of filter also, so we still have opportunity to override it, but by default it will be a common one

Reply:
Added